src/datasets/utils/resources/standard_licenses.tsv

```diff
@@ -19,7 +19,8 @@ Creative Commons Attribution Share Alike 4.0	cc-by-sa-4.0
 Creative Commons Attribution Non Commercial 3.0	cc-by-nc-3.0
 Creative Commons Attribution Non Commercial 4.0	cc-by-nc-4.0
 Creative Commons Attribution No Derivatives 4.0	cc-by-nd-4.0
-Creative Commons Attribution Non Commercial No Derivatives 4.0	cc-by-nc-nd-4.0
+Creative Commons Attribution Non Commercial No Derivatives 4.0 cc-by-nc-nd-4.0
+Creative Commons Attribution Non Commercial Share Alike 2.0 cc-by-nc-sa-2.0
```

Comment:
Please, note that this is a TSV file, but these 2 lines are separated just with a blank character, instead of "\t".